cmd/prcompFormat/testdata/pca.r

```diff
@@ -0,0 +1,17 @@
+library("ggplot2")
+
```

Comment:
If the R isnt necessary for the test files to run, then probably best to keep it in vglDocumentation.

Reply:
I agree. That said, I would support adding a gonum dependency and making some bona fide plotting packages in gonomics.

Reply:
It's a bit more than plotting, in this case R is calculating the actual principal components, the go program just formats the input data from a multiple alignment. I can move the script to vgl

Reply:
Removed in the next commit, and I've added the PCA script to the vglDocumentation repository and added a readme in the testdata folder to direct the user there.

Reply:
gonum also does PCA ;)